### ISSUE TITLE
Split classes into different files

### ISSUE THREAD

**Issue by icc** [h5p/h5p-php-library#13]
To make the code easier to work with, we should put each class into separate file. We should also add or create some sort of default autoloader.

**Comment by falcon-git** [comment]
Should probably follow PHP FIG when doing this?

**Comment by juho-jaakkola** [comment]
I would suggest using composer for autoloading. It's in fact *already* doing it if the library is installed using composer: https://github.com/juho-jaakkola/h5p-php-library/blob/6773f7aad8f95d21c847e12e6ad781e1f8ff26b9/composer.json#L4-L8

Composer currently supports four different autoloading standards: https://getcomposer.org/doc/04-schema.md#autoload

**Comment by icc** [comment]
I saw the autoloading in composer, but I'm not sure as to how it works. Will it generate a file that needs to be included in the plugin/module?

**Comment by juho-jaakkola** [comment]
Yes. When you run composer, it automatically creates `vendor/autoload.php` file, that you then include in the plugin script.

**Comment by davericher** [comment]
I ended up just cutting them up and using Laravels IoC and a service provider to put them all together since 90% of the classes are singletons
